graph/telemetry/istio/istio.go

```diff
@@ -742,11 +742,11 @@ func buildNodeTrafficMap(cluster, namespace string, n *graph.Node, o graph.Telem
 
 		switch o.Rates.Tcp {
 		case graph.RateReceived:
-			metrics = []string{"istio_tcp_received_bytes_total"}
```

Comment:
Above the switch stmt can we add:

// L4 telemetry is backwards, see https://github.com/istio/istio/issues/32399